schema/me/conversation.js

```diff
@@ -74,6 +77,28 @@ export const ConversationParticipantType = new GraphQLObjectType({
   },
 })
 
+export const ConversationResponderType = new GraphQLObjectType({
```

Comment:
I think we should still have a `ConversationParticipantType` that combines `ConversationInitiatorType` and `ConversationResponderType`. 

`reply_to_impulse_ids` can then be an optional input field in `UpdateConversation` mutation.